Source/Core/VideoBackends/Vulkan/VKScheduler.cpp

```diff
@@ -0,0 +1,144 @@
+// Copyright 2022 Dolphin Emulator Project
```

Comment:
If I understand correctly this is imported from another project? If so, please credit them somewhere in the header, e.g. "Imported from XXX" or "Inspired by XXX"